src/openscm_runner/run.py

```diff
@@ -11,11 +11,28 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def _check_out_config(out_config, climate_models_cfgs):
+    if out_config is not None:
+        unknown_models = set(out_config.keys()) - set(climate_models_cfgs.keys())
+        if unknown_models:
```

Comment:
Why not silently ignore? Say we had a script to run MAGICC and Fair, but for testing purposes, we wanted to just run Fair. We would have to keep the out_config and climate_models_cfgs in lockstep. Not a huge problem and very much a style preference, but I think L83 is sufficient.

Reply:
Yep good point. I'll add some extra logging around line 83 so debugging is easier (i.e. to make it not completely silent) but I agree this is probably overkill so will remove.

Reply:
I think my logic here was that if you have a key in `out_config` which isn't in `climate_models_cfgs`, something is obviously wrong. I hadn't thought about the case where you only want to run one so this is a good point.

Reply:
Just pushed updates, see what you think